### ISSUE TITLE
Just wanted to say thanks for putting this together!

### ISSUE THREAD

**Issue by averyvery** [magimetal/Synthwave-X-Jetbrains#1]
I recently had to switch from VSCode to IntelliJ and dearly missed the Synthwave 84 theme. Your adaptation is great, thank you for working on it!

**Comment by timothyfarmer** [comment]
YES! Thanks! I've been looking for a Synthwave 84 theme for Jetbrains! 💯

**Comment by podling0ne** [comment]
Yes thanks a looot . I just tweak and put #1D1D1D for backgroudn and gutter to fit my material darker appearance.
Thanks you rocks :)
